.github/workflows/samples-aspnet-fastenpoints-server.yaml

```diff
@@ -0,0 +1,34 @@
+name: Samples C# .Net 8 FastEndpoints Server
+
+on:
+  push:
+    paths:
+      - samples/server/petstore/aspnetcore-8.0/**
+      - samples/server/petstore/aspnetcore-8.0-*/**
+  pull_request:
+    paths:
+      - samples/server/petstore/aspnetcore-8.0/**
+      - samples/server/petstore/aspnetcore-8.0-*/**
```

Comment:
you will need to update the above 4 lines (path) referencing fastendpoints path mentioned below in order to trigger the CI to run

Reply:
Fixed